kafka-observ-lib/config.libsonnet

```diff
@@ -1,16 +1,17 @@
 {
   local this = self,
   filteringSelector: 'job!=""',
-  groupLabels: ['kafka_cluster'],
```

Comment:
I'm not 100% on this change. `cluster` is a reserved label for use with kubernetes clusters, which is why we changed this to `kafka_cluster` in the past to support multi-cluster kafka separately from kubernetes clusters.